src/prognostic_equations/edmfx_closures.jl

```diff
@@ -206,78 +226,193 @@ function mixing_length(
     ᶜstrain_rate_norm,
     ᶜPr,
     ᶜtke_exch,
+    scale_blending_method,
 )
 
     FT = eltype(params)
+    eps_FT = eps(FT)
+
+
     turbconv_params = CAP.turbconv_params(params)
+    sf_params = CAP.surface_fluxes_params(params) # Businger params
+
     c_m = CAP.tke_ed_coeff(turbconv_params)
     c_d = CAP.tke_diss_coeff(turbconv_params)
     smin_ub = CAP.smin_ub(turbconv_params)
     smin_rm = CAP.smin_rm(turbconv_params)
     c_b = CAP.static_stab_coeff(turbconv_params)
     vkc = CAP.von_karman_const(params)
 
-    # compute the maximum mixing length at height z
-    l_z = ᶜz - z_sfc
+    # MOST stability function coefficients
+    most_a_m = sf_params.ufp.a_m # Businger a_m
+    most_b_m = sf_params.ufp.b_m # Businger b_m
+    most_g_m = CAP.coefficient_b_m_gryanik(params)  # Gryanik b_m
```

Comment:
It's annoying we have to do it this way.

Reply:
Would it help not to have b_m overloaded in parameters (for Businger and Gryanik)? That's confusing anyway, and if it helps here, it may be worth changing in parameters and SurfaceFluxes

Reply:
This is a workaround to avoid more significant refactoring of SurfaceFluxes and/or ClimaAtmos. The SurfaceFlux/Parameter interface in CA is generally written to support either Businger or Gryanik. It would be nice to more easily access both, I can open an issue for it.